tests/test_suite_common.sh

```diff
@@ -43,7 +43,7 @@ execute_test_suite() {
 execute_tests_daemon() {
     # Run confd as a background process.
     echo "Running confd as background process"
-    confd -confdir=/etc/calico/confd -log-level=debug >$LOGPATH/logd1 2>&1 &
+    BGP_LOGSEVERITYSCREEN="debug" confd -confdir=/etc/calico/confd >$LOGPATH/logd1 2>&1 &
```

Comment:
extremely minor nit:  do we need these quotes?

Reply:
We don't, but I've already typed them . . .